redis-cache.php

```diff
@@ -50,6 +54,23 @@ public function __construct() {
 
     }
 
+    // https://core.trac.wordpress.org/ticket/31245
+    // https://github.com/tillkruss/redis-cache/issues/58
+    public function maybe_clear_alloptions_cache( $option ) {
+
+        if ( defined( 'WP_REDIS_IGNORED_GROUPS' ) && in_array( 'options' , WP_REDIS_IGNORED_GROUPS ) ) {
+            return;
+        }
+
+        if ( ! wp_installing() ) {
+            $alloptions = wp_load_alloptions();
+
+            if ( array_key_exists( $option, $alloptions ) ) {
+                wp_cache_delete( $option, 'options' );
```

Comment:
Why are you only deleting the option itself and not the "alloptions" cache? How does this fix the race condition?